integration/spark/app/src/test/java/io/openlineage/spark/agent/DatabricksIntegrationTest.java

```diff
@@ -174,8 +211,10 @@ public void testWriteReadFromTableWithLocation() {
             .get();
 
     // assert input and output are the same
-    assertThat(outputDataset.getNamespace()).isEqualTo(outputDataset.getNamespace());
-    assertThat(inputDataset.getName()).isEqualTo(inputDataset.getName());
+    // TODO: this assertions are not working
+    // https://github.com/OpenLineage/OpenLineage/issues/2543
```

Comment:
Please include what's the difference between those datasets now either here in the comment or in the issue.